src/common/utxobased/engine/types.ts

```diff
@@ -42,13 +42,15 @@ export const asUtxoSignMessageOtherParams = asObject({
   publicAddress: asString
 })
 
+const asOutputSort = asValue('bip69', 'targets')
+
 export type UtxoSpendInfoOtherParams = ReturnType<
   typeof asUtxoSpendInfoOtherParams
 >
 export const asUtxoSpendInfoOtherParams = asObject({
   enableRbf: asOptional(asBoolean),
   forceChangeAddress: asOptional(asString),
-  outputSort: asOptional(asValue('bip69', 'targets'), 'bip69'),
+  outputSort: asOptional(asOutputSort, 'bip69'),
```

Comment:
Added the message: "When the asValue cleaner gets embedded inside asObject, it will often infer string instead of the specific literal type. Moving the asValue cleaner to its own line allows TypeScript to infer the exact string literals, instead of the string catch-all."